src/Engine.sol

```diff
@@ -401,20 +494,28 @@ contract Engine is
         external
         override
     {
-        // ensure account exists (i.e. owner is not the zero address)
-        /// @notice this does not check if the caller is the account owner
-        if (PERPS_MARKET_PROXY.getAccountOwner(_accountId) == address(0)) {
-            revert AccountDoesNotExist();
```

Comment:
**Question:** Why does this check no longer need doing?

Reply:
I was only slightly convinced by macro that it'd be useful. after further consideration, it just seems like wasteful hand-holding